### DISCUSSION TITLE
Unable to restore purchase from previous version of SD MAID PRO

### DISCUSSION THREAD

**Discussion by Axenx-get** [d4rken-org/sdmaid-se#1424] [Q&A] [answered]
good morning I had purchased and installed the SD MAID PRO version on an old smartphone with android 12. now I purchased a new device with android 14 and it suggested installing the SE version. unfortunately, I was unable to restore the purchase in any way. can you help me please? thanks

**Comment by MeloriTensei** [comment, marked as answer]
Hi, please understand that SD Maid 1 Pro is separate from SD Maid 2/SE. And you will need a new license for the SE version.

SD Maid 1 has lived long enough to see the end of it's time.
